### PR TITLE
added model descriptions to results

### PR DESCRIPTION
**Pull request by andrewkern**
trying to patch https://github.com/popsim-consortium/manuscript/issues/95

### CODE REVIEW

**Comment by andrewkern** [comment]
also should say that i took a pass through the methods for model IDs as in https://github.com/popsim-consortium/manuscript/issues/42 and it looks good to me.

**Comment by jeromekelleher** [comment]
LGTM, merging.

File: popSimManu.tex [hunk 1]
```diff
@@ -337,21 +337,27 @@ \subsection*{The \stdpopsim Catalog}
...
+components, a three-population out-of-Africa model from \cite{ragsdale2019models}
+which includes archaic admixture, a complex model of ancient eurasian admixture from \cite{kamm2019efficiently},
+and a synthetic model of oscillating population size from \cite{schiffels2014inferring}.
 Together these models
 contain features believed to have widespread impacts in real data (e.g. bottlenecks, population growth,
 admixture) and are therefore highly pertinent in the context of method development.
 
 For non-human genomes we have have implemented two demographic histories for
-\emph{Drosophila melanogaster} and one each from \emph{A. thaliana} and \emph{E. coli}.
+\emph{Drosophila melanogaster}, three from \emph{A. thaliana}, an one each from \emph{P. abelii} and \emph{E. coli}.
```
Comment (agladstein) [inline popSimManu.tex:348]: *and

Comment (agladstein) [inline popSimManu.tex:348]: Is there one E. coli model? it's not in the Catalog.

**Review by agladstein — COMMENTED** [review]
Other than E. coli model not being listed in the Catalog, looks correct to me.

(Maybe that E. coli model does exist and it's an issue with the Catalog?)

File: popSimManu.tex (hunk 1 above)
Comment (jeromekelleher) [reply popSimManu.tex:348]: It's got the null single population model that all species have

**Pull request merged**